### ISSUE TITLE
DeliverySettings not returned in Members for MembersListCall

### ISSUE THREAD

**Issue by plusworxchris** [googleapis/google-api-go-client#602]
I am using v0.29.0 of the admin/directory/v1 package. When I get the results from a MembersListCall the DeliverySettings field is always an empty string even though I know that the field has been set.

If I do a MembersGetCall for individual members, the DeliverySettings field is returned with its correct value. I'm not sure what's going on here.

**Comment by codyoss** [comment]
I don't think this is an issue with the generated client, it sounds more like something at the service layer. I recommend reaching out to the service team with one of the suggestions on their [support page](https://developers.google.com/admin-sdk/directory/support).

**Comment by plusworxchris** [comment]
Thanks for your response. I searched the issue tracker and there are 2 open issues for this: 116535610 and 122446589. I've starred both but there are only 3 stars for each issue so I don't hold out much hope of a fix any time soon.

While I realise there are priorities when dealing with these issues, I can't imagine this particular issue would be too difficult to fix given that the MembersGetCall returns the correct value. The MembersListCall must surely just be a variation on that theme. Anyway, at least I now know that it's not just me being foolish.
